### DISCUSSION TITLE
Enter removes styles from ngx-editor

### DISCUSSION THREAD

**Discussion by Akhis-World** [sibiraj-s/ngx-editor#412] [Q&A] [answered]
When enter text in new line, the pre selected styles are removed. Any help would be highly appreciable.

**Comment by sibiraj-s** [comment, marked as answer]
Enter creates a new paragraph, hence the styles are not carried through. If you still want a line break within try `SHIFT+ENTER`. Hope that solves the issue.

**Reply by Akhis-World** [reply]
Yes, that works. Thank you!
But any solution to make the paragraphs carry styles ?

**Reply by sibiraj-s** [reply]
You might need to write a custom prosemirror plugin/keymap for that and override the default.

[a comment by an automated account is not shown]
